.prettierrc.yml

```diff
@@ -0,0 +1,9 @@
+trailingComma: all
+printWidth: 120
```

Comment:
https://prettier.io/docs/en/options.html#print-width gives good reasons as to why this should be 80, but it's not up to me to deceide.

Reply:
I'm ok with 80 actually...it makes GH side-by-side diff work nicely in all cases that I use it :) But... maybe one thing at a time?